### PR TITLE
Update pytest-dash to v2.0.0

### LINKED ISSUE #180: Update to pytest-dash v2.0.0
Main breaking change is the removal of `pytest-selenium` support, the selenium driver is now included on the [`dash_threaded`/`dash_subprocess` fixtures](https://pytest-dash.readthedocs.io/en/latest/usage.html#dash-threaded-example). The `--driver` argument is now `--webdriver`, you can set in `pytest.ini` config file.

### PR DESCRIPTION
**Pull request by Bachibouzouk**
Closes #180 

## Description of changes
Change `tests/requirements.txt`
Fix deprecated references to `pytest_dash.utils`

## Before merging
- [ ] I have gone through the [code review checklist](https://github.com/plotly/dash-component-boilerplate/blob/master/%7B%7Bcookiecutter.project_shortname%7D%7D/review_checklist.md)
- [ ] I have completed all of the [steps to take before merging](https://github.com/plotly/dash-bio/blob/master/.github/before_merging.md)
- [ ] I know how to [deploy to DDS](https://github.com/plotly/dash-bio/blob/master/.github/deploying_to_dds.md)

### CODE REVIEW

**Comment by Bachibouzouk** [comment]
Right now the webdriver seems to not work properly 
`pytest --webdriver Chrome tests/test_volcano_plot.py` raises a `pytest` error : `_pytest.config.exceptions.UsageError: --driver must be specified`

**Comment by T4rk1n** [comment]
> --driver must be specified

That is a pytest-selenium error, I removed support for that.

**Comment by Bachibouzouk** [comment]
removing `selenium` as an argument of the test functions and replacing `selenium` by `dash_threaded.driver` does work, thanks @T4rk1n for your help :)

File: tests/test_volcano_plot.py [hunk 1]
```diff
@@ -203,12 +208,11 @@ def assert_callback(p_value, nclicks, input_value):
     )
 
 
-def test_effect_size_line_input_value(dash_threaded, selenium):
+def test_effect_size_line_input_value(dash_threaded):
     """Modifies the effect_size line value."""
```
Comment (Bachibouzouk) [inline tests/test_volcano_plot.py]: This test fails because the function converting the string to an array raises a ValueError

**Comment by mkcor** [comment]
Thank you, @Bachibouzouk!

**Pull request merged**